libraries/core_libs/network/src/tarcap/packets_handlers/transaction_packet_handler.cpp

```diff
@@ -26,18 +33,23 @@ inline void TransactionPacketHandler::process(const PacketData &packet_data, con
   transactions.reserve(transaction_count);
 
   for (size_t tx_idx = 0; tx_idx < transaction_count; tx_idx++) {
-    const auto transaction = std::make_shared<Transaction>(packet_data.rlp_[tx_idx].data().toBytes());
+    std::shared_ptr<Transaction> transaction;
+
+    try {
+      transaction = std::make_shared<Transaction>(packet_data.rlp_[tx_idx].data().toBytes());
+    } catch (const Transaction::InvalidSignature &e) {
```

Comment:
I now it is not part of this task, but `InvalidSignature` should be the case, when you verifying transaction. Maybe we should introduce new exception like InvalidFormat

Reply:
I did create low priority task for this https://github.com/Taraxa-project/taraxa-node/issues/1572